packages/jolokia_input/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.2.0"
+  changes:
+    - description: Updating the Kibana Version to 8.8.0
```

```suggestion
    - description: Update the Kibana Version to 8.8.0
```

Making it consistent with the change made for `packages/statsd_input/changelog.yml` and also because of this: https://github.com/elastic/integrations/pull/6333#discussion_r1206407368